src/transactions/transactions.controller.spec.ts

```diff
@@ -387,4 +390,15 @@ describe('TransactionsController', () => {
       });
     });
   });
+
+  describe('GET /transactions/deposit_address', () => {
+    it('retuns deposit address from config', async () => {
+      const { body } = await request(app.getHttpServer())
+        .get('/transactions/deposit_address')
```

Comment:
Now that we have the deposit controller, since these are related to deposits, should we move this to `deposits/address`?

Reply:
yea I'm ok with that